HTMLSerializer.js

```diff
@@ -112,6 +126,30 @@ var HTMLSerializer = class {
   }
 
   /**
+   * Takes an html node of type Node.TEXT_NODE, and add its text content with
+   *     all characters properly escaped to |this.html|.
+   * @param {Node} node The text node.
+   */
+   // TODO(sfine): Take care of attribute value normalization:
+   // https://developers.whatwg.org/the-iframe-element.html#the-iframe-element
+  processText(node) {
```

Comment:
Philip and I weren't able to find any.  You can do the following:

var div = document.createElement('div');
div.appendChild(document.createTextNode('<text that needs escaping&&">');
var text = div.innerHTML

But it doesn't escape all of the characters that need to be.